### ISSUE TITLE
Gene (CDK2) to Pathway?

### ISSUE THREAD

**Issue by cbizon** [NCATS-Tangerine/cam-kp-api#256]
Query:
```
query={
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "id": "NCBIGene:1017",
                    "category": "biolink:Gene"
                },
                "n1": {
                    "category": "biolink:Pathway"
                }
            },
            "edges": {
                "e01": {
                    "subject": "n0",
                    "object": "n1",
                    "predicate": "biolink:related_to"
                }
            }
        },
        'knowledge_graph': {'nodes': {}, 'edges': {}},
        'results': []
    }
}
```

This is returning a 400. But the response.content says:
b'Request to https://stars-app.renci.org timed out after 45000 ms'

**Comment by balhoff** [comment]
This query works if you use `NCBIGENE` instead of `NCBIGene` (as specified by biolink-model: https://github.com/biolink/biolink-model/issues/725).

However you don't get any results with `Pathway`. `BiologicalProcess` gives results:

```json
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "id": "NCBIGENE:1017",
                    "category": "biolink:Gene"
                },
                "n1": {
                    "category": "biolink:BiologicalProcess"
                }
            },
            "edges": {
                "e01": {
                    "subject": "n0",
                    "object": "n1",
                    "predicate": "biolink:related_to"
                }
            }
        }
    }
}
```

**Comment by rajshruti18** [comment]
Closing this issues because it's being addressed here: #301
